core-plugins/docs/AzureBlobStore-batchsource.md

```diff
@@ -17,7 +17,7 @@ Properties
 **referenceName:** This will be used to uniquely identify this source for lineage, annotating metadata, etc.
 
 **path:** The path on Microsoft Azure Blob Storage to use as input. The path uses filename expansion (globbing) to read
-files. (Macro-enabled)
+files. The path must strat with `wasb://` or `wasbs://`. (Macro-enabled)
 
 **account:** The Microsoft Azure Blob Storage account to use. This is of the form 
 `mystorageaccount.blob.core.windows.net`, where `mystorageaccount` is the Microsoft 
```

Comment:
@YanAtCask I am not sure if we should do that because Azure also supports custom domains on http protocol https://docs.microsoft.com/en-us/azure/storage/storage-custom-domain-name#register-a-custom-domain. Not sure if it is also supported for `wasb` protocol.